summernote/summernote
# Update image popover with new icons

#### What does this PR do?

- Previously, image popover used the same icons of text alignments. But floating is not working as like as alignment.
- I added icons for `float: left`, `float: right` and for rollback.
- I renamed `imageSizeXXX` as `resizeXXX` to match the names of handlers and language property names.
- I renamed `alignXXX` as `floatXXX` to have the right names.

#### Where should the reviewer start?

- Please see the changes.

#### How should this be manually tested?

- Build and run. You have to run `npm run dist` to build new icons. Add an image into the summernote area to see the image popover.

#### Any background context you want to provide?

- There is no `float: justify`, but we'd used `alignJustify` for `float: none`. So I changed it and also fixed related stuffs.

#### What are the relevant tickets?

- https://github.com/summernote/summernote/issues/2956

#### Screenshot (if for frontend)

<img width="739" alt="screen shot 2018-12-06 at 3 49 52 pm" src="https://user-images.githubusercontent.com/579366/49569601-bcdfad00-f977-11e8-91f5-b087e0caab9b.png">

### Checklist
